src/enums/document-types.ts

```diff
@@ -0,0 +1,40 @@
+export enum DocumentClassEnum {
```

Comment:
So far in this project, we've been naming enums like this: `DocumentClasses`.

Are you OK with this?